templates/docs/base/typography.md

```diff
@@ -103,9 +107,14 @@ Our type scale consists of 5 font sizes, expressed as rems (root em units). For
       <p class="p-heading--1" style="font-size: 2.5rem; line-height: 3rem">H1 heading</p>
     </td>
   </tr>
+  <tr>
+    <td>64px</td>
+    <td>Ad hoc display headings for important bespoke pages on small and medium screens.</td>
```

Comment:
why the inline styles?

Reply:
In order to make sure that the right size is displayed, independent of the screen you are currently on. But I am now realising, that we need to do it both ways actually. So also need to set inline styles for the large screen version.

Reply:
I guess the question is what do we want to present here: all the available size options, regardless of screen size - even if we never use them on same screen size (for example, both sizes of h2 heading, both sizes of display heading) - I guess 8 options? - this requires examples with inline styles.

Or do we want to show how many options we have on given screen size (5?): so list 1 row for h2 heading, 1 row for display heading, but maybe mention that their size depends on screen size? (and keep examples responsive)